### DISCUSSION TITLE
Missing utility function in equation (4) in the Botorch paper

### DISCUSSION THREAD

**Discussion by Mehdishishehbor** [meta-pytorch/botorch#1147] [Q&A] [answered]
Hi,

I have a question regarding an equation in BOTROCH paper, BOTORCH: A Framework for Efficient Monte-Carlo Bayesian Optimization, and I was wondering if someone could help me with it. 

In equation (1) I can see utility function, a, while in equation (4), there seems to be a missing utility function. Why is it missing? Is it a typo or a reason behind not involving utility function in equaion (4). Thank you in advance for your help.

equation (1)
<img width="259" alt="image" src="https://user-images.githubusercontent.com/94387791/160686672-28fb8e3a-fd4b-4e38-b89f-68c718b6a256.png">

equation (4)
<img width="266" alt="image" src="https://user-images.githubusercontent.com/94387791/160683808-1931cb33-fff0-46c4-9515-6702ece38982.png">

**Comment by Balandat** [comment, marked as answer]
Hi, (1) is a general form of MC acquisition functions, whereas (4) is specific to KG. It so happens that in KG we maximize the posterior mean, so the utility function “a” here is just the identity. You could use a different utility function in a lookahead strategy as well, but then that wouldn’t be the classic Knowledge Gradient.

**Reply by Mehdishishehbor** [reply]
Thank you Balandat for your help.
